src/NetAnalyzers/UnitTests/Microsoft.CodeQuality.Analyzers/QualityGuidelines/AvoidMultipleEnumerationsTests.cs

```diff
@@ -1284,7 +1283,28 @@ public void Sub(IEnumerable<int> h)
     }
 }";
             await VerifyCS.VerifyAnalyzerAsync(code);
+        }
+
+        [Fact]
+        public async Task TestInvocationLocalAssignmentWithDeferredMethodCall()
```

Comment:
Would it be possible to add the `vbnet` side of the tests you have added?

Reply:
Sure, I will first add all the vb side tests in this PR. For the rest of the tests, I can submit another PR

Reply:
Oops, so I really find operation differences between VB and C#, working a on fix..

Reply:
Actually the VB side fix is not trivia, I will create another PR for that. In the meantime I will mark this as draft